### PR TITLE
ms ss rg collector update

### PR DESCRIPTION
**Pull request by mksample**

### CODE REVIEW

File: collectors/collector-json.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
-const collector_json = function (data, params) {
+const collector_json = function (datas, params) {
     // convert JSON data string to object form
-    if (typeof data === 'string' || data instanceof String) {
-        data = JSON.parse(data);
+
+    for (let [i, data] of datas.entries()) {
```
Comment (sarahsemkow) [inline collectors/collector-json.js:4]: is datas no longer an array here?

Comment (mksample) [reply collectors/collector-json.js:4]: entries() transforms the array into tuples containing an index and data

File: collectors/collector-json.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+        if (typeof data === 'string' || data instanceof String) {
+            data[i] = JSON.parse(data);
```
Comment (sarahsemkow) [inline collectors/collector-json.js:6]: did you mean to just do data = JSON.parse(data) ?

Comment (mksample) [reply collectors/collector-json.js:6]: lol yes I did

**Review by sarahsemkow — APPROVED** [review]
looks good now!

**Review by b-hsieh — APPROVED** [review]

**Pull request merged**